exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -132,10 +132,10 @@ func createLabelSet(resource pdata.Resource, labels pdata.StringMap, externalLab
 		return true
 	})
 
-	labels.Range(func(key string, value string) bool {
+	attributes.Range(func(key string, value pdata.AttributeValue) bool {
 		l[key] = prompb.Label{
 			Name:  sanitize(key),
-			Value: value,
+			Value: value.StringVal(), // TODO(codeboten): Decide what to do with non-string attributes.
```

Comment:
We should probably stringify non-string attributes. Unclear what encoding to use though for non-primitive types. Perhaps json-string or something like that since that's also what Otel spec recommends for exporting array attributes when arrays are not natively supported in the target format: https://github.com/open-telemetry/opentelemetry-specification/blob/7cef355a15c4430aa323d155206a2945fcd67a05/specification/common/common.md#attributes

Reply:
Sure i can open a separate issue to track that work

Reply:
Please open the issue to not forget :)

Reply:
Done https://github.com/open-telemetry/opentelemetry-collector/issues/3815